$NetBSD: patch-arch_arm_dts_Makefile,v 1.5 2019/06/05 05:43:15 thorpej Exp $

Add device tree files for:
- Banana Pi BPI-M2 Zero
- Banana Pi BPI-P2 Zero

--- arch/arm/dts/Makefile.orig	2019-06-02 17:45:33.522314609 -0700
+++ arch/arm/dts/Makefile	2019-06-02 17:46:19.143587602 -0700
@@ -365,6 +365,8 @@ dtb-$(CONFIG_MACH_SUN8I_A83T) += \
 	sun8i-a83t-cubietruck-plus.dtb \
 	sun8i-a83t-tbs-a711.dts
 dtb-$(CONFIG_MACH_SUN8I_H3) += \
+	sun8i-h2-plus-bananapi-m2-zero.dtb \
+	sun8i-h2-plus-bananapi-p2-zero.dtb \
 	sun8i-h2-plus-libretech-all-h3-cc.dtb \
 	sun8i-h2-plus-orangepi-r1.dtb \
 	sun8i-h2-plus-orangepi-zero.dtb \
